src/playground/sample_data_tooling/sample_data_generator/README.md

```diff
@@ -0,0 +1,187 @@
+            '865-50-6891'
+            ~~~
+            - If you wanted timestamps generated to simulate real-time data ingestion, `unix_time` is the only format currently supported. To configure timestamps for this purpose, see the Ingestion tool.
+        - `args` (list): The arguments for `kind`. If you want to specify key word arguments, use a dictionary inside the list.
+            - *ex:* `choose_field("float", [2, 3, True])` generates a positive float with two left digits and three right digits. See [pyfloat](https://faker.readthedocs.io/en/master/providers/faker.providers.python.html) for more information on arguments
+            ~~~
+            >>> choose_field("float", [2, 3, True])
+            `24.387'
+            ~~~
+            - *ex:* `choose_field("file_extension", ["image"])` taken directly from [file_extension](https://faker.readthedocs.io/en/master/providers/faker.providers.file.html) generates an image file format (jpg, png, etc.)
+            ~~~
+            >>> choose_field("file_extension", ["image"])
+            'jpeg'
+            ~~~
+            - *ex:* `choose_field("file_name", [{"category": "audio"}])` taken directly from [file_name](https://faker.readthedocs.io/en/master/providers/faker.providers.file.html) generates a file name with the `category` argument set to `audio`, which means audio filenames will be generated
+            ~~~
+            >>> choose_field("file_name", [{"category": "audio"}])
+            'career.flac'
+            ~~~ 
+                - **Note:** Make sure the dictionary is *inside* the list for `choose_field()` to properly parse arguments.
+    - **Returns:** 
+        - Depending on the data type you specify, `choose_field()` may generate `int`, `float`, `string`, `dicts`, and more
+- `generate_data()`: Given an input type (listed below) and a mapping flag, the function generates a document in the specified format
+    - **Arguments:**
+        - `mapping` (boolean): Whether or not the `data_template` is a JSON string/dict [mapping](https://opensearch.org/docs/latest/opensearch/mappings/). Currently, only explicit mapping is supported. This value is `True` by default.
+        - `data_template` (JSON string, dictionary, string): The data template of the document to be generated. There are four types of input:
+            - `JSON string/dict mapping`: A mapping of the document to be generated. This format allows for users who specify an [explicit mapping](https://opensearch.org/docs/latest/opensearch/mappings/) for their index settings to then just copy the `mappings` value as an argument to `generate_data()`. `mapping` must be set to `True`.
+
+            For this example, say we want to generate bike data, like for instance, prices (between $100 and $1000) and manufacturers. The data template, then, would be as shown below:
+            ~~~
+            >>> # JSON Mapping as a dict
+            >>> bike_template = {
+            ...    "properties": {
```

Comment:
this should be a whitespace, right ?

Reply:
Yes, this code block is the Python shell. That is why there are `...` and `>>>` in the code block.